distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -136,6 +136,70 @@ public void testBasicRestAccess() throws Exception {
+        OpenSearchSourceProperties openSearchProperties = new OpenSearchSourceProperties(
+                OPENSEARCH_SOURCE_ID);
+        openSearchProperties.put("username", "admin");
+        openSearchProperties.put("password", "admin");
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchProperties);
+
+        CswSourceProperties cswProperties = new CswSourceProperties(CSW_SOURCE_ID);
+        cswProperties.put("username", "admin");
+        cswProperties.put("password", "admin");
+        createManagedService(CswSourceProperties.FACTORY_PID, cswProperties);
+
+        waitForFederatedSource(OPENSEARCH_SOURCE_ID);
+        waitForFederatedSource(CSW_SOURCE_ID);
+
+        String openSearchQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + OPENSEARCH_SOURCE_ID;
+        given().auth().basic("admin", "admin").when().get(openSearchQuery).then().log().all()
+                .assertThat().statusCode(equalTo(200)).assertThat().body(containsString("myTitle"));
+
+        String cswQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + CSW_SOURCE_ID;
+        given().auth().basic("admin", "admin").when().get(cswQuery).then().log().all().assertThat()
+                .statusCode(equalTo(200)).assertThat().body(containsString("myTitle"));
+
+        //Negative tests
+        String unavailableCswSourceId = "Unavailable Csw";
+        cswProperties = new CswSourceProperties(unavailableCswSourceId);
+        cswProperties.put("username", "bad");
+        cswProperties.put("password", "auth");
+        createManagedService(CswSourceProperties.FACTORY_PID, cswProperties);
+
+        String cswQueryUnavail = SERVICE_ROOT + "/catalog/query?q=*&src=" + unavailableCswSourceId;
+        given().auth().basic("admin", "admin").when().get(cswQueryUnavail).then().log().all()
+                .assertThat().statusCode(equalTo(500));
+
+        /*
+        //Negative tests TODO: DDF-1444 Negative OpenSearchSource BasicAuth tests currently do not work
+        String unavailableOpenSourceId = "Unavailable OpenSearchSource";
+
+        OpenSearchSourceProperties openSearchUnavailProp = new OpenSearchSourceProperties(
+                unavailableOpenSourceId);
+        openSearchUnavailProp.put("username", "bad");
+        openSearchUnavailProp.put("password", "auth");
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchUnavailProp);
+        waitForFederatedSource(unavailableOpenSourceId);
+
+        String unavailableOpenSearchQuery =
+                SERVICE_ROOT + "/catalog/query?q=*&src=" + unavailableOpenSourceId;
+
+        given().auth().basic("admin", "admin").when().get(unavailableOpenSearchQuery).then().log()
+                .all().assertThat().statusCode(equalTo(200)).assertThat()
+                .body(not(containsString("myTitle")));
+
+        String unavailableCswQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + unavailableCswSourceId;
+        */
+
+        configureRestForAnonymous();
```

Comment:
These two lines are having issues. We're having an issue deleting the metacard we ingest. We thought we had resolved the issue by making sure to reconfigure rest for anonymous before deleting the card, and it had worked once but now fails. Does configuring for anonymous require some time that we should wait for before attempting to delete the metacard? I'd like feedback from @stustison and @pklinef on this.